### PR TITLE
changed version and sha256

### PR DESCRIPTION
**Pull request by grenaud**
I made a minor modification to the Makefile but I need to change the version number.

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/libgab/meta.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
   name:  {{ name }}
   version: {{ version }}
 
 source:
   url: "https://github.com/grenaud/{{ name }}/archive/{{ version }}.tar.gz"
-  sha256: 9f1cfb358bb42efa21ee3e84f9122080041c0cf9987ebdea563c5992bb85994a
+  sha256: 04bd27813e3c001f64d06bee4524502e99f0f80e4a0882ffcead8770b7b2a2da
 
 build:
   number: 0
```
Comment (apeltzer) [inline recipes/libgab/meta.yaml:13]:
```suggestion
  number: 1
```

**Review by apeltzer — APPROVED** [review]
As the bot one was merged - I just added a usggestion to bump build to 1 to get your makefile patch in 👍🏼

**Comment by apeltzer** [comment]
Once this is passed through CI, can simply let the bot do the job by typing in `@bioconda-bot please merge` @grenaud

File: recipes/libgab/meta.yaml [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
   version: {{ version }}
 
 source:
   url: "https://github.com/grenaud/{{ name }}/archive/{{ version }}.tar.gz"
-  sha256: 9f1cfb358bb42efa21ee3e84f9122080041c0cf9987ebdea563c5992bb85994a
+  sha256: 04bd27813e3c001f64d06bee4524502e99f0f80e4a0882ffcead8770b7b2a2da
 
 build:
-  number: 0
+  number: 1
```
Comment (apeltzer) [inline recipes/libgab/meta.yaml:13]:
```suggestion
  number: 0
```

File: recipes/libgab/meta.yaml (hunk 1 above)
Comment (apeltzer) [inline recipes/libgab/meta.yaml:13]:
```suggestion
  number: 1
```

**Comment by apeltzer** [comment]
Ok, got this past the linting now @grenaud, once it is finished testing (which it will pass pretty sure): I'll merge it then afterwards.

**Comment by apeltzer** [comment]
@bioconda-bot please merge

**Comment by grenaud** [comment]
@apeltzer  thanks a bunch!

**Pull request merged**